src/ui-kit/components/Button/Button.jsx

```diff
@@ -0,0 +1,26 @@
+/** @jsxImportSource @emotion/react */
+import { StyledButton } from './Button.styled';
+import { memo } from 'react';
+
+function DefaultButton({
+  startIcon = null,
+  isDisabled = false,
+  endIcon = false,
+  label = 'Button',
+  handleClick,
+  ...props
```

Comment:
Please change structure: label, click, isDisabled, startIcon, endIcon, withoutBackground, fontSize ...props
And add withoutBackground, fontSize